dynawo/sources/Models/Modelica/Dynawo/UsersGuide/NamingConventions.mo

```diff
@@ -165,6 +160,6 @@ e.g tInteger, t0...</li>
 
 <h3>Preferred view</h3>
 
-<p> Always set a preferred view to your model such as it will be opened by default in this view by Modelica-based environments:</p><p></p><ul><li>text view for equation-based models</li><li>diagram view for input/output or block models</li><li>info view for documentation</li></ul><p></p></body></html>"));
+<p> Always set a preferred view to your model such as it will be opened by default in this view by Modelica-based environments:</p><p></p><ul><li>text view for equation-based models</li><li>diagram view for input/output or block models</li><li>info view for documentation</li></ul><div><h3>Blank lines and indentation</h3></div><div>Follow these rules regarding blank lines and indentation:</div><div><ul><li>One blank line between the \"within\" and the copyright</li><li>One blank line between the copyright and the model name</li><li><b>No</b> blank line between the model name and the first import</li><li>One blank line between the imports and the extends</li><li>One blank line between the extends and the parameters</li><li>One blank line between the parameters and the connectors/inputs/outputs</li><li>One blank line between the connectors/input/outputs and the blocks</li><li>One blank line between the blocks and the initial parameters</li><li>One blank line between the initial parameters and the protected section</li><li><b>No</b> blank line between the \"protected\" word and the first protected element</li><li>One blank line between the protected parameters and the protected init parameters</li><li>One blank line between the protected init parameters and protected variables</li><li>One blank line between the protected variables and the equation section</li><li><b>No</b> blank line between the \"equation\" word and the first equation</li><li>One blank line between equations and a when equation or an if equation</li><li>One blank line before annotations</li><li><b>No</b> blank line between annotations and end of model</li></ul><div>Two spaces must be used for indentation.</div></div><p></p></body></html>"));
```

Comment:
I would prefer a blank line between the model name and the first import line, but I'll do as you wish.

Reply:
To be consistent with the rest of the library and what we do with protected and equation sections